### ISSUE TITLE
int(Min, Max) shrinks to 0 instead of Min

### ISSUE THREAD

**Issue by nickva** [triqng/triq#65]
Noticed that int(Min, Max) domain shrinks to 0 instead of Min. Is that expected?

For example:
```
125>  triq_dom:sampleshrink(triq_dom:int(5, 10)).
8
[7]
[6]
[5]
[4]
[3]
[2]
[1]
[0]
ok
```

If this is a bug, it looks 0 should be replaced with Min in this fragment:

https://github.com/triqng/triq/blob/master/src/triq_dom.erl#L607-L609

**Comment by tuncer** [comment]
Thanks, nice catch. My guess is that this is close enough to the correct behavior that it went unnoticed. Would you like to submit a patch with a test case?

**Comment by nickva** [comment]
Hi @tuncer 

Thanks for taking a look. I gave it a try.  Let me know if it looks good or needs improvements.

**Comment by KrzysiekJ** [comment]
Fixed in b4ce70c9e292c2e576e3076cdaa6151352417fde.
